src/main/webapp/__tests__/createGameElements.test.js

```diff
@@ -0,0 +1,128 @@
+// Provides more matchers like `toBeOneOf`
+require('jest-extended'); 
+// module path is relative to 'src/main/webapp/'
+const { trueOrFalseGrid, multipleChoiceGrid, titleGrid, questionElement } = require('scripts/createGameElements.js');
+
+// `describe` blocks can be used to group related tests together.
+describe('createGameGrids', () => {
+  // `test` is a single unit test
+  test('Is the true or false grid appearing', () => {
```

Comment:
Be wary of writing tests that closely mirror the code they are testing. These tests will break if there are any changes to the HTML output by the functions. 

A more reliable test might do this:
1) append `trueOrFalseGrid(0)` to `document.body`
2) query for list of checkbox inputs.
3) `expect` that the checkbox inputs have the right ID attributes ("question0checkbox0" and "question0checkbox1").